src/main.js

```diff
@@ -76,9 +76,15 @@ export function generateCompress(isEncode) {
  * @param {Object} options
  * @param {string} [options.body]
  */
-export const fetchToCurl = (url, options) => {
+const fetchToCurl = (requestInfo, requestInit) => {
+  const url = typeof requestInfo === 'string' ? requestInfo : requestInfo.url;
+  const options = typeof requestInfo === 'string' ? requestInit : requestInfo;
+  const fetchHeaders = options.headers ?
```

Comment:
This will fail if someone only supplies a url as string. (options will be `ùndefined` in that case)